### PR TITLE
Policy Details Update to look like the rest

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2021-08-13 at 9 54 15 AM

### CODE REVIEW

File: portal-ui/src/screens/Console/Policies/PolicyDetails.tsx [hunk 1]
```diff
@@ -178,6 +203,9 @@ const PolicyDetails = ({
   const [filterGroups, setFilterGroups] = useState<string>("");
   const [loadingGroups, setLoadingGroups] = useState<boolean>(true);
 
+  const [buckets, setBuckets] = useState<Bucket[]>([]);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/PolicyDetails.tsx:206]: This var is not needed, please remove

File: portal-ui/src/screens/Console/Policies/PolicyDetails.tsx [hunk 2]
```diff
@@ -287,19 +317,85 @@ const PolicyDetails = ({
     elementItem.includes(filterGroups)
   );
 
+  const closeAddModalAndRefresh = (refresh: boolean) => {
+    if (refresh) {
+      setLoadingUsers(true);
+      setLoadingGroups(true);
+      setLoadingPolicy(true);
+    }
+  };
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/PolicyDetails.tsx:326]: This function is not needed, please remove

**Review by bexsoft — CHANGES_REQUESTED** [review]
Beside of that, LGTM

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by adfost — APPROVED** [review]

**Pull request merged**